electron-app/src/server/websites/artconomy/artconomy.service.ts

```diff
@@ -229,12 +229,28 @@ export class Artconomy extends Website {
     const files = [
       submission.primary,
       ...(submission.additional || []).filter(
-        f => !f.ignoredAccounts!.includes(submissionPart.accountId),
+        (f) => !f.ignoredAccounts!.includes(submissionPart.accountId),
       ),
     ];
 
-    const maxMB: number = 99;
-    files.forEach(file => {
+    const tags = [...submissionPart.data.tags.value];
+    if (submissionPart.data.tags.extendDefault) {
+      tags.push(...defaultPart.data.tags.value);
+    }
+
+    if (tags.length < 5) {
+      problems.push(
+        'You must have at least 5 tags. Think about the following: ' +
+          'sex/gender (required, if character), ' +
+          'species (ditto), kinks/fetishes (ditto), genre, subject matter, ' +
```

Comment:
I'm going to merge this. But I am going to tweak out the kink/fetish part. Just a personal preference